### PR TITLE
Update to netcoreapp3.1

### PR DESCRIPTION
**Pull request by tabman83**

### CODE REVIEW

File: src/EShopWorld.Tools/EShopWorld.Tools.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (mszymeshop) [inline src/EShopWorld.Tools/EShopWorld.Tools.csproj]: Should we also change the version of the package and its release notes?

Comment (tabman83) [reply src/EShopWorld.Tools/EShopWorld.Tools.csproj]: Right, thanks.

File: src/EShopWorld.Tools/EShopWorld.Tools.csproj [hunk 2]
```diff
@@ -11,24 +11,24 @@
     <ToolCommandName>dotnet-esw</ToolCommandName>
     <RuntimeIdentifiers>win-x64</RuntimeIdentifiers>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <Version>1.1.31</Version>
+    <Version>1.2.0</Version>
```
Comment (mszymeshop) [inline src/EShopWorld.Tools/EShopWorld.Tools.csproj]: @VaclavK can you advise? To me, this is a breaking change and the cli should be moved to 2.0.0

Comment (VaclavK) [reply src/EShopWorld.Tools/EShopWorld.Tools.csproj]: either is fine.. this is not used externally and as long as the version is new the pipeline will perform the cli update

for our sake, let's go with major

Comment (mszymeshop) [reply src/EShopWorld.Tools/EShopWorld.Tools.csproj]: @tabman83 So it should say 2.0.0

**Pull request merged**